irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,239 @@
+        $("#username").val(sessionStorage.IRODS_username);
+    }
+    if(sessionStorage.IRODS_password) {
+        $("#password").val(sessionStorage.IRODS_password);
+    }
+    if(sessionStorage.IRODS_host) {
+        $("#host").val(sessionStorage.IRODS_host);
+    }
+    if(sessionStorage.IRODS_port) {
+        $("#port").val(sessionStorage.IRODS_port);
+    }
+    if(sessionStorage.IRODS_zone) {
+        $("#zone").val(sessionStorage.IRODS_zone);
+    }
+});
+
+$('#btn-select-irods-file').on('click',function(){
+    $('#res_type').val($('#resource-type').val());
+    $('#file_struct').children().remove();
+    if (sessionStorage.IRODS_signininfo) {
+        $("#irods_content_label").text(sessionStorage.IRODS_username);
+        $('#root_store').val(sessionStorage.IRODS_datastore);
+    }
+    var store = $('#root_store').val();
+    // Setting up the view tab
+    $('#file_struct').attr('name',store);
+    $('#irods_view_store').val(store);
+    // loading file structs
+    var parent = $('#file_struct');
+    get_store(store,parent,0);
+    enable_settings();
+});
+
+function get_store(store, parent, margin){
+    $.ajax({
+        mode: "queue",
+        url: '/irods/store/',
+        async: true,
+        type: "POST",
+        data: {
+            store: store,
+            user: sessionStorage.IRODS_username,
+            password: sessionStorage.IRODS_password,
+            zone: sessionStorage.IRODS_zone,
+            port: sessionStorage.IRODS_port,
+            host: sessionStorage.IRODS_host
+        },
+        success: function (data, status) {
+            data = jQuery.parseJSON(data);
+            if (data[0].length == 0 && data[1].length == 0) {
+                $(parent).append("<div class='file' style='margin-left:" + margin + "px;'></div>");
+            }
+            else {
+                $.each(data[1],function(i,v) {
+                    $(parent).append("<div class='folder' id='irods_folder_" + v + "' name='" + store + "/" + v + "' style='margin-left:" + margin + "px;'><img src='/static/img/folder.png' width='15' height='15'>&nbsp; " + v + "</div>");
+                });
+
+                $.each(data[0],function(i,v) {
+                    $(parent).append("<div class='file' id='irods_file_" + v + "' name='" + store + "/" + v + "' style='margin-left:" + margin + "px;'><img src='/static/img/file.png' width='15' height='15'>&nbsp; " + v + "</div>")
+                    $('.file').on('click',function(e){
+                        $('.file').css('background','');
+                        $('.file').removeClass('selected');
+                        $(e.target).css('background','#cecece');
+                        $(e.target).addClass('selected');
+                        set_datastore($(this).parent('div')[0], 0)
+                        return false
+                    });
+                });
+            }
+        },
+
+        error: function(status) {
+            console.log(status);
```

Comment:
you should use `console.error()` here which shows the message as an error and, on some browser dev tools, will show a traceback to find the problem.